pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -1465,9 +1465,11 @@ private void makeStreamConsumer(String reason) {
     if (_partitionGroupConsumer != null) {
       closePartitionGroupConsumer();
     }
-    _segmentLogger.info("Creating new stream consumer, reason: {}", reason);
+    _segmentLogger.info("Creating new stream consumer {} for partition group {} , reason: {}", _clientId,
```

Comment:
nit: looks like `_clientID` is already `streamTopic + "-" + _partitionGroupId`. Do we need to again add partition group to the log?

Reply:
ah you are right. 
The log wasn't very clear when I simply used the `clientId` as the reader had to know that a clientId is made up of `streamTopic` and `partitionGroupId`.  So I included the "for partition group" part. I can change it back to use only `_clientId`

Reply:
fixed